pkgs/os-specific/linux/wpa_supplicant/wheel_dbus_access.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/wpa_supplicant/dbus/dbus-wpa_supplicant.conf b/wpa_supplicant/dbus/dbus-wpa_supplicant.conf
+index e81b495f4..d1622c765 100644
+--- a/wpa_supplicant/dbus/dbus-wpa_supplicant.conf
++++ b/wpa_supplicant/dbus/dbus-wpa_supplicant.conf
+@@ -2,16 +2,11 @@
+  "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+  "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+ <busconfig>
+-        <policy user="root">
++        <policy group="wheel">
+                 <allow own="fi.w1.wpa_supplicant1"/>
+ 
+                 <allow send_destination="fi.w1.wpa_supplicant1"/>
+                 <allow send_interface="fi.w1.wpa_supplicant1"/>
+                 <allow receive_sender="fi.w1.wpa_supplicant1" receive_type="signal"/>
+         </policy>
+-        <policy context="default">
+-                <deny own="fi.w1.wpa_supplicant1"/>
+-                <deny send_destination="fi.w1.wpa_supplicant1"/>
```

Comment:
If I'm reading https://dbus.freedesktop.org/doc/dbus-daemon.1.html correctly then the default for "sending" and ''owning" is already "deny", so these should be no-ops.

Why would you want to remove them here? And if you want that, shouldn't that be proposed upstream?